### ISSUE TITLE
create_eog_epochs does not detect the maximum signal during blink events

### ISSUE THREAD

**Issue by stfnrpplngr** [mne-tools/mne-python#4892]
Hi,
using the `mne.preprocessing.create_eog_epochs()` function does result in weird solutions. The epochs center is not the center of the peak. I would guess it has something to do with the filtering during blink detection. What is your opinion?

Average EOG plotting channels Fp1 and Fp1
![average_eog_notaligned](https://user-images.githubusercontent.com/20129752/35393559-94122518-01e5-11e8-8ad1-2a695b94a66b.png)

EOG epochs plotting channels Fp1 and Fp2
![eog_epochs_not on peaks center](https://user-images.githubusercontent.com/20129752/35393587-ab994e3c-01e5-11e8-8536-6af69bab2399.PNG)

**Comment by agramfort** [comment]
yes it's not detected properly but it should not be a pb to estimate an SSP
